lxd/instance/vmqemu/vm_qemu.go

```diff
@@ -2898,46 +2954,56 @@ func (vm *vmQemu) ExpiryDate() time.Time {
 	return time.Time{}
 }
 
-func (vm *vmQemu) Path() string {
+// Path returns the instance's path.
+func (vm *VMQemu) Path() string {
 	return storagePools.InstancePath(vm.Type(), vm.Project(), vm.Name(), vm.IsSnapshot())
 }
 
-func (vm *vmQemu) DevicesPath() string {
+// DevicesPath returns the instance's devices path.
+func (vm *VMQemu) DevicesPath() string {
 	name := project.Prefix(vm.Project(), vm.Name())
 	return shared.VarPath("devices", name)
 }
 
-func (vm *vmQemu) ShmountsPath() string {
+// ShmountsPath returns the instance's shared mounts path.
+func (vm *VMQemu) ShmountsPath() string {
 	name := project.Prefix(vm.Project(), vm.Name())
 	return shared.VarPath("shmounts", name)
 }
 
-func (vm *vmQemu) LogPath() string {
+// LogPath returns the instance's log path.
+func (vm *VMQemu) LogPath() string {
 	name := project.Prefix(vm.Project(), vm.Name())
 	return shared.LogPath(name)
 }
 
-func (vm *vmQemu) LogFilePath() string {
-	return filepath.Join(vm.LogPath(), "lxvm.log")
+// LogFilePath returns the instance's log path.
+func (vm *VMQemu) LogFilePath() string {
+	return filepath.Join(vm.LogPath(), "lxvm.log") // tomp TODO is this correct?
```

Comment:
@stgraber I noticed that this file name doesn't look quite right. Should it be lxdvm.log or just vm.log?

Reply:
I suspect we'll want `qemu.log` but this is unused so not that important.